### DISCUSSION TITLE
Welcome to dlprimitives Discussions!

### DISCUSSION THREAD

**Discussion by artyom-beilis** [artyom-beilis/dlprimitives#4] [Announcements]
##  Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IIIBlueberry** [comment]
What do you think on Porting DLPrimitives Op to Pytorch Vulkan backend?

Pytorch has experimental Vulkan backend originally for Android devices, it can also be used on desktop but there is not a lot of Op available
https://pytorch.org/tutorials/prototype/vulkan_workflow.html

**Reply by artyom-beilis** [reply]
Currently pytorch/dlprimitives integration backend already implements much more operations than Vulkan backend: https://github.com/artyom-beilis/pytorch_dlprim

You already can do both training and inference on major vision networks:.

**Comment by rh-id** [comment]
I want to help but I don't have a lot of cpp knowledge, the best I could help is testing this on my device, any straight forward test script I could try?

I'm interested in pytorch integration, any test for that? 

Also OpenCV seemed to have full support on OpenCL not sure if it will help you just FYI

 https://opencv.org/opencl/

**Comment by artyom-beilis** [comment]
> he best I could help is testing this on my device, any straight forward test script I could try?

Sure run tests (there are benchmarks and unit test

> I'm interested in pytorch integration, any test for that?

Use it try different networks different setups you aware of and see what happends
